### ISSUE TITLE
Build Error - CMakeList.txt and LSL.java

### ISSUE THREAD

**Issue by ryohajika** [labstreaminglayer/liblsl-Android#3]
Hi. I found the CMakeList.txt have two lines referring non-existing boost source files. I could build the whole project successfully after I commented out these. Is anybody able to check whether these two lines still need for the project or not?

https://github.com/labstreaminglayer/liblsl-Android/blob/7b5ac43c399a07cd2608abcce45818dcd645d05f/AndroidStudio/CMakeLists.txt#L59-L60

**Comment by ryohajika** [comment]
I think this one also needs to be refined from `load` to `loadLibrary`...

https://github.com/labstreaminglayer/liblsl-Java/blob/67791ef8eb45b059306cb0c26849cf5c81e63980/src/edu/ucsd/sccn/LSL.java#L1110

**Comment by tstenner** [comment]
> Hi. I found the CMakeList.txt have two lines referring non-existing boost source files. I could build the whole project successfully after I commented out these. Is anybody able to check whether these two lines still need for the project or not?

They aren't needed any more, I missed those because the newer gradle builds use the CMakeLists.txt in the main repository.

> I think this one also needs to be refined from load to loadLibrary...

The Android code uses JNA 5.1 where `loadLibrary` is deprecated.

**Comment by ryohajika** [comment]
On the latter, if I leave the line as original (`inst = (dll)Native.load("lsl", dll.class);`), error `error: cannot find symbol method load(String,Class<dll>)` get arose. The whole project will be built successfully after I change it to `loadLibrary`.

**Comment by mgrivich** [comment]
Tristan,
   Whenever changing the Android or Java code, _please_ download the entire labstreaminglayer project fresh (wiping all build products), and rebuild the ReceiveString and SendString examples from within Android Studio on Windows. It is broken once again. It looks like the new version of JNA is not being downloaded automatically, in addition to Hzikajr's legitimate comments.

**Comment by tstenner** [comment]
> Whenever changing the Android or Java code, please download the entire labstreaminglayer project fresh (wiping all build products), and rebuild the ReceiveString and SendString examples from within Android Studio on Windows. It is broken once again.

That's what I did after my last change 5 weeks ago for PR #2. A fresh clone doesn't build, because nobody's bothered to update the submodule references in the main repo after you merged the PR yet. I just did that and it's building and running the SendStringMarkers example.

>  It looks like the new version of JNA is not being downloaded automatically

In the liblsl-Android project I got a different error message related to Gradle, but after I pulled the updated repo Gradle downloaded JNA and unpacked the libraries without me having to do anything. For liblsl-Java it doesn't do that automatically, because the binaries in JNA are in an android specific package format and I haven't had time to look how to specify a dependency for plain Java where the binaries are handled correctly.

>  in addition to Hzikajr's legitimate comments.

For the first part, we could delete the now obsolete CMakeLists.txt or remove the two lines. What's your opinion on it?  
For the second part we could either revert to the deprecated loadLibrary call or change the instructions in liblsl-Java to download JNA 5.1.
